starport/pkg/chaincmd/chaincmd.go

```diff
@@ -336,14 +336,14 @@ func (c ChainCmd) GentxCommand(
 		commandGentx,
 	}
 
-	if c.sdkVersion.Is(cosmosver.StargateZeroFourtyAndAbove) {
+	if c.sdkVersion.Is(cosmosver.StargateZeroFortyToZeroFortyTwo) {
```

Comment:
Seems like there is bug here. This the condition here should be true for any chain that ZeroFourtyAndAbove.

What do you think about implementing a `cosmosver.Version.GTE(Version)`func and use it here. It could basically iterate through `cosmosver.Versions`and check against the given version, knowing that `cosmosver.Versions` is already a sorted list.

Reply:
nice, I like the idea!